lab-raziyeh/index.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+const main = require('./lib/fs-files.js')
+main();
```

Comment:
@Raziyehbazargan Nice index file, I love the separation of concerns in your file structure!